src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double GROW_INDEX = 1.5;
+    private static final String INDEX_OUT_OF_BOUNDS_EXEPTION = "This index "
+            + "out of bounds: ";
+    private static final String NO_SUCH_ELEMENT_EXEPTION = "Element not found in list ";
+    private T[] elementData = (T[]) new Object[DEFAULT_CAPACITY];
```

Comment:
I think declaration of T[] elementData in class is good but
private T[] elementData = (T[]) new Object[DEFAULT_CAPACITY]; - should be in constructor, i think
but code working fine

Reply:
By default, .length of ArrayList is 10.
You can declare another size, while init new ArrayList.